coalib/bearlib/aspects/Spelling.py

```diff
@@ -1,72 +1,66 @@
-from coalib.bearlib.aspects import Root
+from coalib.bearlib.aspects import Root, Taste
 
 
 @Root.subaspect
 class Spelling:
-    """
-    This aspect describes spelling of your source code.
+    """`Spelling <https://en.wikipedia.org/wiki/Spelling>_` refers to how words
```

Comment:
I dont see how linking to wikipedia is particularly useful here, and it is very unusual compared with your other docstrings, and pushes this docstring over the one line limit.